core/invoker/build.gradle

```diff
@@ -8,11 +8,16 @@ distDocker.dependsOn ':common:scala:distDocker', 'distTar'
 
 repositories {
     mavenCentral()
+    maven {
+        url "https://dl.bintray.com/akka/maven/"
```

Comment:
alpakka bits are in central, I don't think this is needed
https://mvnrepository.com/artifact/com.lightbend.akka/akka-stream-alpakka-file_2.11/0.14